src/1_Iterations/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Given a positive integer N, Computes the length of its longest binary gap. 
+ * Returns 0 if N doesn't contain a binary gap
+ * @param {number} N 
+ */
+function solution(N) {
+  if (!Number.isInteger(N) || N < 0)
```

Comment:
I don't think this is part of the instructions. It says 
```
that, given a positive integer N, returns the length of its longest binary gap.
```
So no need for the check.